src/disambiguate-series-statements.js

```diff
@@ -0,0 +1,218 @@
+import createDebugLogger from 'debug';
+import {fieldHasSubfield, fieldToString, nvdebug} from './utils';
+import {MARCXML} from '@natlibfi/marc-record-serializers';
+import {Error} from '@natlibfi/melinda-commons';
+import {default as createNatlibfiSruClient} from '@natlibfi/sru-client';
+
+//const {default: createNatlibfiSruClient} = natlibfiSruClient;
+
+const debug = createDebugLogger('@natlibfi/marc-record-validators-melinda:disambiguate-series-statements');
+
+const ELECTRONIC = 1;
+const PRINTED = 2;
+const UNDEFINED = 0;
```

Comment:
Could undefined just be undefined or is there some reason that it needs to map to a numerical value?

Reply:
Probably. I didn't want to check whether recordType is an integer or undefined, so I made them all integers.